server.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/ubuntu-core/identity-vault/service"
 )
 
+const sshKeysPath = "/.ssh/authorized_keys"
```

Comment:
That is the default location for the ssh files for Ubuntu systems (and most other OS's), so it didn't seem necessary to make it configurable.